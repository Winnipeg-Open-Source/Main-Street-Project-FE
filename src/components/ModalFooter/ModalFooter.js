import React from 'react';
import PropTypes from 'prop-types';
import { Button } from 'pcln-design-system';
import ButtonGroup from 'components/ButtonGroup';

function ModalFooter ({ className, onClose }) {
    return (
        <ButtonGroup className={className} justifyContent='flex-end'>
            <Button color='background.light' onClick={onClose}>Close</Button>
            <Button>Confirm</Button>
        </ButtonGroup>
    );
}

ModalFooter.displayName = 'ModalFooter';

ModalFooter.propTypes = {
    className: PropTypes.string,
    onClose: PropTypes.func,
};

ModalFooter.defaultProps = {
    className: '',
};

export default ModalFooter;
