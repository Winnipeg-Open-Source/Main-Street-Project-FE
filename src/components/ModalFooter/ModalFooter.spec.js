import React from 'react';
import renderWithTheme from 'tests/helpers/renderWithTheme';
import ModalFooter from './ModalFooter';

describe ('ModalFooter', () => {
    it ('renders without crashing', () => {
        const { asFragment } = renderWithTheme(<ModalFooter />);

        expect(asFragment()).toMatchSnapshot();
    });
});
