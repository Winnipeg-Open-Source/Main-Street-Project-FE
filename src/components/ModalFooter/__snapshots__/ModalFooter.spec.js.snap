// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pbJYR hNzbLI sc-pkfPC fCxHK"
  >
    <div
      class="sc-AxjAm sc-AxiKw sc-psDhf efsLuN"
    >
      <button
        class="sc-oUOMp hMxAoF"
        color="background.light"
      >
        Close
      </button>
      <button
        class="sc-oUOMp gVrnEU"
        color="primary"
      >
        Confirm
      </button>
    </div>
  </div>
</DocumentFragment>
`;
